### PR TITLE
docs: fix sealos run cmd for the single cluster

### PR DESCRIPTION
**Pull request by wyfeng001**

### CODE REVIEW

**Comment by cuisongliu** [comment]
It seems that the new version of sealos does not need the --single parameter

**Comment by bxy4543** [comment]
> It seems that the new version of sealos does not need the --single parameter

Yes, the readme should be updated after the new version is released.

**Review by bxy4543 — APPROVED** [review]

**Comment by bxy4543** [comment]
> [CLA assistant check](https://cla-assistant.io/labring/sealos?pullRequest=2967) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/labring/sealos?pullRequest=2967) before we can accept your contribution.
> 
> **Wang Yanfeng** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/labring/sealos?pullRequest=2967) it.

Please sign CLA for Github account @wyfeng001 .

File: README.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ $ sealos run labring/kubernetes:v1.25.0 labring/helm:v3.8.2 labring/calico:v3.24
 > Single host
 
 ```bash
-$ sealos run labring/kubernetes:v1.25.0 labring/helm:v3.8.2 labring/calico:v3.24.1
+$ sealos run labring/kubernetes:v1.25.0 labring/helm:v3.8.2 labring/calico:v3.24.1 --single
```
Comment (fanux) [inline README.md:104]: New version don't need this args any more

**Comment by bxy4543** [comment]
The readme will be modified to a new version (4.2.0), which will not require this parameter, close this pr

**Pull request closed without merging**